### PR TITLE
fix(katana): range iterating on blocks for events

### PR DESCRIPTION
**Pull request by glihm**
Little fix to ensure that events from the latest block are actually retrieved.

### CODE REVIEW

File: crates/katana/core/src/sequencer.rs [hunk 1]
```diff
@@ -423,7 +423,7 @@ impl Sequencer for KatanaSequencer {
             .ok_or(SequencerError::BlockNotFound(to_block))?;
 
         let mut events = Vec::new();
-        for i in from_block.0..to_block.0 {
+        for i in from_block.0..to_block.0 + 1 {
```
Comment (kariy) [inline crates/katana/core/src/sequencer.rs:426]:
```suggestion
        for i in from_block.0..=to_block.0 {
```

**Review by kariy — CHANGES_REQUESTED** [review]

File: crates/katana/core/src/sequencer.rs (hunk 1 above)
Comment (glihm) [reply crates/katana/core/src/sequencer.rs:426]: Wow, didn't know that was possible. That's even better. :) Thanks @kariy.

**Comment by kariy** [comment]
nice catch!

**Review by glihm — COMMENTED** [review]
Changed to the new syntax, thank you @kariy.

**Review by kariy — APPROVED** [review]

**Pull request merged**